src/Microsoft.ApplicationInsights.AspNetCore/project.json

```diff
@@ -100,13 +100,10 @@
             }
         },
         "netstandard1.6": {
-            "imports": [
-                "dotnet5.6",
-                "portable-dnxcore50+net451+net45+win8+wp8+wpa81"
-            ],
-            "dependencies": {
-                "System.Net.NameResolution": "4.0.0"
-            }
+          "dependencies": {
+            "NETStandard.Library": "1.6.0",
```

Comment:
Okay, leave the * in the 2.2.0-beta4-* for now then since we'll need to reference the build of the base I'll make today in order to get that.
I'll see about just calling the base SDK update for netstandard1.5 beta5 so there is not a problem with old private debug builds of beta4 that won't have it getting referenced.